src/routes/forgot-password/forgot-password.js

```diff
@@ -1,19 +1,15 @@
 import React, { Component } from 'react';
-import PropTypes from 'prop-types';
 import { Grid } from 'semantic-ui-react';
 import { ForgotPasswordForm } from 'components/forms';
 import { SubmissionError } from 'redux-form';
 import { userService } from 'api';
+import { push } from 'react-router-redux';
 
 export class ForgotPassword extends Component {
-  static propTypes = {
-    push: PropTypes.func.isRequired
-  };
-
   // replace as soon as auth is in place
   onSubmit = (values) => {
     return userService.resetPassword(values.email)
-      .then(() => this.props.push('login'))
+      .then(() => push('login'))
```

Comment:
does this work without actually being connected to redux? feels like it would disturbs the browser history, but if it works, then great.